services/data/db_utils.py

```diff
@@ -14,17 +14,17 @@ def aiopg_exception_handling(exception):
     body = {"err_msg": err_msg}
     if isinstance(exception, psycopg2.IntegrityError):
         if "duplicate key" in err_msg:
-            return DBResponse(response_code=409, body=json.dumps(body))
+            return DBResponse(response_code=409, body=body)
         elif "foreign key" in err_msg:
-            return DBResponse(response_code=404, body=json.dumps(body))
+            return DBResponse(response_code=404, body=body)
         else:
-            return DBResponse(response_code=500, body=json.dumps(body))
+            return DBResponse(response_code=500, body=body)
     elif isinstance(exception, psycopg2.errors.UniqueViolation):
-        return DBResponse(response_code=409, body=json.dumps(body))
+        return DBResponse(response_code=409, body=body)
     elif isinstance(exception, IndexError):
         return DBResponse(response_code=404, body={})
     else:
-        return DBResponse(response_code=500, body=json.dumps(body))
```

Comment:
Unless I'm missing something, before this fix body was doubly-encoded here https://github.com/Netflix/metaflow-service/blob/master/services/metadata_service/api/utils.py#L24

Reply:
This is somewhat duplicative of #184 (or the idea is for that to be the cleanup PR). I need to change things in the client too and we need to support both ways of doing things. But yes, this is planned.